util/fpga/cw310_loader.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import argparse
+
+import cw_spiflash
+
+import chipwhisperer as cw
+from chipwhisperer.capture.targets.CW310 import CW310
+
+
+def main(argv):
+
+    bergparser = argparse.ArgumentParser(
```

Comment:
What is a "bergparser"...?

Reply:
Good catch, changed the variable name to `parser`. It was a remainder of the original script (the board is also called Bergen board).